src/components/SimpleCard/SimpleCard.ts

```diff
@@ -45,6 +45,7 @@ const SIMPLE_CARD_RENDERED_EVENT = "@nosto/SimpleCard/rendered"
 @customElement("nosto-simple-card", { observe: true })
 export class SimpleCard extends ReactiveElement {
   @property(String) handle!: string
+  @property(Number) variantId?: number
```

Comment:
What if the value is passed directly to this attribute? Does it propagate to the `variant-selector`?

Reply:
currently not